lib/vrl/stdlib/src/chunks.rs

```diff
@@ -0,0 +1,151 @@
+use ::value::Value;
+use vrl::prelude::*;
+
+fn chunks(value: Value, chunk_size: Value) -> Resolved {
+    let bytes = value.try_bytes()?;
+    let chunk_size = chunk_size.try_integer()?;
+
+    if chunk_size < 1 {
+        return Err(r#""chunk_size" must be at least 1 byte"#.into());
```

Comment:
This message uses a `byte` unit, where the one below is just a number. It'd be nice to keep them consistent.